services/madoc-ts/src/frontend/site/pages/search.tsx

```diff
@@ -1,146 +1,68 @@
-import { LocaleString } from '../../shared/components/LocaleString';
-import { Pagination } from '../../shared/components/Pagination';
-import {
-  SearchFilterCheckbox,
-  SearchFilterContainer,
-  SearchFilterItem,
-  SearchFilterItemCount,
-  SearchFilterItemList,
-  SearchFilterLabel,
-  SearchFilterSection,
-  SearchFilterSectionTitle,
-  SearchFilterTitle,
-  SearchFilterToggle,
-} from '../../shared/components/SearchFilters';
-import { SearchResults, TotalResults } from '../../shared/components/SearchResults';
-import { AddIcon } from '../../shared/icons/AddIcon';
-import { useSiteConfiguration } from '../features/SiteConfigurationContext';
-import { useSearch } from '../hooks/use-search';
-import { useSearchFacets } from '../hooks/use-search-facets';
+import { Slot } from '../../shared/page-blocks/slot';
 import { useSearchQuery } from '../hooks/use-search-query';
-import { ButtonRow, TinyButton } from '../../shared/navigation/Button';
+import { StaticPage } from '../features/StaticPage';
+import { SearchPageFilters } from '../features/SearchPageFilters';
+import { AppliedFacets } from '../features/AppliedFacets';
+import { Heading1 } from '../../shared/typography/Heading1';
+import { SearchPagination } from '../features/SearchPagination';
+import { SearchPageResults } from '../features/SearchPageResults';
+import { DisplayBreadcrumbs, useBreadcrumbs } from '../../shared/components/Breadcrumbs';
+import { useRouteContext } from '../hooks/use-route-context';
+import { LocaleString } from '../../shared/components/LocaleString';
 
 export const Search: React.FC = () => {
-  const { t } = useTranslation();
-  const [{ resolvedData: searchResponse, latestData }, displayFacets, isLoading] = useSearch();
-  const { rawQuery, page, fulltext, appliedFacets } = useSearchQuery();
-  const {
-    project: { showSearchFacetCount },
-  } = useSiteConfiguration();
-  const {
-    inQueue,
-    applyFacet,
-    clearSingleFacet,
-    queueSingleFacet,
-    dequeueSingleFacet,
-    isFacetSelected,
-    applyAllFacets,
-    clearAllFacets,
-    setFullTextQuery,
-  } = useSearchFacets();
+  const { rawQuery, page, fulltext } = useSearchQuery();
+
+  const breads = useBreadcrumbs();
+  const isGlobal = !!breads.subpage;
+
+  const getHeading = () => {
+    if (breads.manifest) return breads.manifest?.name;
+    else if (breads.collection) return breads.collection.name;
+    else {
+      return breads.project?.name;
+    }
+  };
 
   return (
-    <>
-      <DisplayBreadcrumbs currentPage="Search" />
+    <StaticPage title="Search">
+      <Slot name="common-breadcrumbs">
+        <DisplayBreadcrumbs currentPage="Search" />
+      </Slot>
+
+      <Slot name="search-heading">
+        {isGlobal ? (
+          <Heading1>“{fulltext}” search</Heading1>
+        ) : (
+          <Heading1>
+            search in <LocaleString>{getHeading()}</LocaleString>
+          </Heading1>
+        )}
```

Comment:
Should move that to it's own component - possibly with a way to override the heading.